libyul/AsmAnalysis.cpp

```diff
@@ -135,7 +135,7 @@ vector<YulString> AsmAnalyzer::operator()(Identifier const& _identifier)
 		{
 			bool insideFunction = m_currentScope->insideFunction();
 			size_t stackSize = m_resolver(_identifier, yul::IdentifierContext::RValue, insideFunction);
-			if (stackSize != size_t(-1))
+			if (!m_errorReporter.hasErrors() && stackSize != size_t(-1))
```

Comment:
What happens without this?

Reply:
without this `found` will be set to `true` and finally not setting `m_success` to `false`. probably this can be cleaned up a bit.

Reply:
Please see above.

Reply:
Ah, nice!